CHANGELOG.md

```diff
@@ -1,5 +1,80 @@
 # [Versions](https://mui.com/versions/)
 
+## v6.0.0-alpha.9
+
+<!-- generated comparing v6.0.0-alpha.8..next -->
+
+_May 29, 2024_
+
+A big thanks to the 23 contributors who made this release possible.
+This release was mostly about 🐛 bug fixes and 📚 documentation improvements.
```

Comment:
Should the deprecations be added in highlights ?

Reply:
I didn't add it as a highlight since I saw they were never highlighted before!

Reply:
I wouldn't add them honestly